core/templates/pages/preferences-page/preferences-page.component.spec.ts

```diff
@@ -97,11 +99,14 @@ describe('Preferences Controller', function() {
     var isRequestTheExpectOne = function(queryParams) {
       return decodeURIComponent(queryParams).match('"update_type":"user_bio"');
     };
+    const preventReloadEventremoveSpy = spyOn(
+      PreventReloadEventService, 'removeListener').and.callThrough();
 
     $httpBackend.expect(
       'PUT', '/preferenceshandler/data', isRequestTheExpectOne).respond(200);
     ctrl.saveUserBio(userBio);
     $httpBackend.flush();
+    expect(preventReloadEventremoveSpy).toHaveBeenCalled();
```

Comment:
Can we do `expect(PreventReloadEventService.removeListener).toHaveBeenCalled()`? [This will look clear in terms of testing the service function getting called.]

Reply:
DONE